index.html

```diff
@@ -39,29 +39,25 @@
                 <p class="videoTitle"> Playing an amazing Video in youtube </p>
                 <div class="row my-0">
                     <div class="col-1">
-                        <img class="youtublogo1 " style="width:50px; height: 50px; border-radius: 50%;"
-                            src="assets/youtublogo.png" alt="youtublogo">
+                        <img class="youtublogo1 " style="width:44px; height: 44px; border-radius: 50%;" src="assets/youtublogo.png" alt="youtublogo">
                     </div>
                     <div class="col-6 text-left">
                         <span style="font-size:12px; color:#fff; display:block;"> Micro Code </span>
                         <span style="font-size:12px; color:#fff; display: inline-block;"> subscriber 40k </span> &nbsp;
                         <button class="btn btn-sm btn-light fw-bold subscribBtn"> subscrib </button>
                     </div>
                     <div class="col-5 text-right text-white mt-1">
-                        <button class="btn btn-sm btn-dark fw-bold rounded-4">
+                        <button class="btn btn-sm btn-dark fw-bold rounded-3">
                             <span style="font-size:18px; color:#fff; display:inline-block;">
-                                <i class="fa fa-thumbs-up" aria-hidden="true"></i> 44
-                            </span> &nbsp; | &nbsp;
+                                <i class="fa fa-thumbs-up" aria-hidden="true"></i> 44 </span> &nbsp; | &nbsp;
                             <span style="font-size:18px; color:#fff; display:inline-block;">
                                 <i class="fa fa-thumbs-down" aria-hidden="true"></i>
                             </span>
                         </button>
-                        <button class="btn btn-sm btn-dark fw-bold rounded-4"
+                        <button class="btn btn-sm btn-dark fw-bold rounded-3"
                             style="font-size:18px; color:#fff; display:inline-block;"> <i class="fa-solid fa-share"></i>
                             share ... </button>
-                        <button class="btn btn-sm btn-dark fw-bold rounded-5"
-                            style="font-size:16px; color:#fff; display:inline-block;">
-                            ... </button>
+                        <button class="btn btn-sm btn-dark fw-bold rounded-5 more-btn">  ... </button>
                     </div>
                 </div>
```

Comment:
You have done a great job so far!👏👏
But it would be better to use css flex box in this section.